### PR TITLE
Added support for tmux xdg config path

### PR DESCRIPTION
**Pull request by jpds**
tmux attempts to read `~/.config/tmux/tmux.conf` and displays a permission denied error without this addition.

### CODE REVIEW

File: policy/modules/apps/screen.te [hunk 1]
```diff
@@ -97,6 +97,8 @@ userdom_create_user_pty(screen_domain)
 userdom_setattr_user_ptys(screen_domain)
 userdom_setattr_user_ttys(screen_domain)
 
+xdg_read_config_files(screen_domain)
```
Comment (pebenito) [inline policy/modules/apps/screen.te:100]: Is tmux reading other files under $HOME/.config or is this just for the directory access?

Comment (dsugar100) [reply policy/modules/apps/screen.te:100]: Also, please wrap this in a optional block so xdg module can be disabled and screen still enabled.

Comment (jpds) [reply policy/modules/apps/screen.te]: @pebenito Indeed for directory access, without it I hit an error.

@dsugar100 Done.

Comment (jpds) [reply policy/modules/apps/screen.te]: @pebenito Just tried it with search_dirs() and that works fine.

**Review by pebenito — APPROVED** [review]

**Pull request merged**